pages/api/recipesMeals/[nameMeal].js

```diff
@@ -0,0 +1,43 @@
+import axios from 'axios';
+
+const ingredientsWithMeasure = (ingredients, measure) => {
+  const list = ingredients.reduce((accumulator, currentValue, index) => {
+    const ingredientWithMeasure = currentValue + " " + measure[index];
+    return [...accumulator, ingredientWithMeasure]
+  }, []);
+  return list;
+}
+const itens = (recipeDetails, element) => {
+  const list = Array(19).fill(0).reduce((acc, _currentValue, index) => {
+    if (!recipeDetails[`${element}${index + 1}`]) {
+      return acc
+    }
+    const ingredient = recipeDetails[`${element}${index + 1}`]
+    return [...acc, ingredient]
+  }, []);
+  return list;
+};
+
+export default async function getAllCategories(req, res) {
```

Comment:
should this function be named something else since it is not referring to getting categories?

Reply:
Yes, I will change this name now!